r1cs/gadgets/std/src/pairing/mnt4/mod.rs

```diff
@@ -1,3 +1,14 @@
+/*
+Pairing gadget for MNT4 curves:
+    - MNT4PairingGadget, and
+    - the implementation of the PairingGadget in alignment to its primitive
+      (Ate pairing with flipped Miller loop, using pre-computations).
+
+To do: generic treatment of the sign of the trace  using  ATE_IS_LOOP_COUNT_NEG as well as
```

Comment:
We already did it and merge into development. Merge the last development branch into this one before approving the PR and remove this comment.